examples/deberta/azureml_registry_config.json

```diff
@@ -0,0 +1,97 @@
+{
+    "azureml_client": {
```

Comment:
is this required here? since azureml_client is optional for registry model

Reply:
Good catch! Removed.